File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.ts
function differenceOfSum(nums: number[]): number {
   let elementSum=nums.reduce((a,b)=>a+b,0);
   let numsInDigit=nums.join("").split("");
   let digitSum=numsInDigit.reduce((a,b)=>a+Number(b),0);
   return elementSum - digitSum;
};

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.cs
public class Solution {
    public int DifferenceOfSum(int[] nums) {
        int elementSum=nums.Aggregate((a,b)=>a+b);
        var numsInDigit=string.Join("", nums).ToCharArray();
        int digitSum=0;
        for(int i=0;i<numsInDigit.Length;i++){
            digitSum+=numsInDigit[i]-'0';
        }
        return elementSum-digitSum;
    }
}